inc/classes/subscriber/CDN/RocketCDN/DataManagerSubscriber.php

```diff
@@ -50,7 +50,7 @@ public static function get_subscribed_events() {
 			'wp_ajax_rocketcdn_process_set'    => 'set_process_status',
 			'wp_ajax_rocketcdn_process_status' => 'get_process_status',
 			self::CRON_EVENT                   => 'maybe_disable_cdn',
-			'update_option_' . WP_ROCKET_SLUG  => [ 'maybe_update_api_status', 12, 2 ],
+			'update_option_' . rocket_get_constant( 'WP_ROCKET_SLUG' ) => [ 'maybe_update_api_status', 12, 2 ],
```

Comment:
Now the signs are not aligned anymore :s

Reply:
The sniffers are not triggered on that one, I think the structure of the key throw them off